nornir/core/inventory.py

```diff
@@ -105,6 +105,18 @@ def __contains__(self, value: object) -> bool:
         else:
             return any([value == g for g in self])
 
+    def add(self, group: "Group") -> None:
```

Comment:
Would you mind adding a docstring indicating that `add` will not append the Group to the list if the object is already present?